modules/ROOT/pages/design-helpful-features-text-editor.adoc

```diff
@@ -82,3 +90,84 @@ types:
           application/json:
             type: person  <3>
 ----
+
+== Find Locations Where a Definition is Referenced
+
+You can navigate to locations where a given definition is referenced by right-clicking on the definition and selecting *Go to References*.
+
+For example, the type `person` is declared on line 5 at the first callout in the following example API specification. That type is subsequently used on line 14 at the second callout, and on line 22 at the third callout. If you right-click `person` on line 5 and select *Go to References*, a list of the `person` locations (callouts two and three) appears. Double clicking on a listed location will navigate you to the relevant line.
+
+----
+#%RAML 1.0
+title: Test References
+
+uses:
+  lib: library.raml 
+
+types:
+  person:  <1>
+    description: this is person's description
+    properties:
+      firstname: string
+  manager:
+    type: lib.manager 
+  Alertable: manager | person  <2>
+
+/person:
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            type: person[]  <3>
+----
+
+The action also supports the following:
+
+* RAML resource types and traits
+* RAML type expressions
+* Library aliases
+* References linked in JSON path navigation
+* References to nodes that apply “macros”, such as annotationTypes, securitySchemes, resourceTypes and traits
+
+== Select Code Ranges
+
+“Selection Ranges” allow you to select and to expand the selection of code blocks. 
```

Comment:
GLOBALLY replace "allow" with "enable," per the style ref.

Reply:
Fixed